pkg/base/linuxutils/partitionhelper/partition_helper.go

```diff
@@ -175,8 +174,14 @@ func (p *WrapPartitionImpl) GetPartitionTableType(device string) (string, error)
 // CreatePartition creates partition with name partName on a device
 // Receives device path to create a partition
 // Returns error if something went wrong
-func (p *WrapPartitionImpl) CreatePartition(device, label string) error {
-	cmd := fmt.Sprintf(CreatePartitionCmdTmpl, device, label)
+func (p *WrapPartitionImpl) CreatePartition(device, label, partUUID string, setUUID bool) error {
+	var cmd string
+	switch setUUID {
```

Comment:
Don't use `switch` for two statements, Use `if` here, please.

Reply:
Refactored